includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -966,6 +1009,22 @@ public static function print_validation_errors_meta_box( $post ) {
 		} );
 		</script>
 
+		<?php
+		$has_forced_sanitized = false;
+		foreach ( $validation_errors as $validation_error ) {
+			if ( $validation_error['forced'] ) {
```

Comment:
You're probably aware of this, but I saw notices on an "Invalid AMP Page (URL)" pages:

>( ! ) Notice: Undefined index: forced in /srv/www/loc/public_html/wp-content/plugins/amp/includes/validation/class-amp-invalid-url-post-type.php on line 1015

<img width="1377" alt="amp-invalid-url" src="https://user-images.githubusercontent.com/4063887/40859112-45674d24-65a6-11e8-9578-803f648c224b.png">

This occurred even after clicking "Re-check."

But these could be from errors saved from before this PR. I tried bulk moving "Invalid Pages" to the trash, but this didn't help.

Still, you [mentioned](https://github.com/Automattic/amp-wp/pull/1190#discussion_r192407969) that we don't need backwards compatibility with another version of `1.0`. And the way validation errors are stored is very different from in `0.7`.